server/configuration_test.go

```diff
@@ -0,0 +1,77 @@
+package server
+
+import (
+	"os"
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+func TestNewConfiguration(t *testing.T) {
+	// Set environment variables
+	os.Setenv(DARE_PORT, "8080")
```

Comment:
`t.Setenv` is preferable in tests since it takes care of setting back the variables once the test ends.

Reply:
ok thanks for the comment!